src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -110,14 +110,15 @@ public function load(array $configs, ContainerBuilder $container)
     }
 
     /**
-     * Enables JSON-LD and Hydra support.
+     * Enables JSON-LD and Hydra and Swagger support.
      *
      * @param XmlFileLoader $loader
      */
     private function enableJsonLd(XmlFileLoader $loader)
     {
         $loader->load('jsonld.xml');
         $loader->load('hydra.xml');
+        $loader->load('swagger.xml');
```

Comment:
Swagger support is not related to JSON-LD support. You can import the file directly here: https://github.com/Simperfit/core/blob/119af86fdd0ba61090320d73055b3e87d50c2e2a/src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php#L84